playbooks/check_pulp_dependency_versions.yml

```diff
@@ -34,26 +34,35 @@
     - name: Check pulp_ansible version and set scaling flag
       when: _api_pods.resources | length > 0
       block:
+        - name: Wait for API pod to be fully ready
+          kubernetes.core.k8s_info:
+            api_version: v1
+            kind: Pod
+            namespace: "{{ ansible_operator_meta.namespace }}"
+            name: "{{ _api_pods.resources[0].metadata.name }}"
+            wait: true
+            wait_condition:
+              type: Ready
+              status: "True"
+            wait_timeout: 300
+          when: _api_pods.resources | length > 0
```

Comment:
I think we can remove that condition because it's already there on at the block level